.github/workflows/vulnerability_scan_subworkflow.yml

```diff
@@ -0,0 +1,82 @@
+name: Vulnerability Scan
+
+on:
+  workflow_call:
+    inputs:
+      ref:
+        required: true
+        type: string
+    secrets:
+      SNYK_TOKEN:
+        required: true
+  workflow_dispatch:
+    inputs:
+      ref:
+        required: true
+        type: string
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - label: oss
```

Comment:
This looks to be https://github.com/hazelcast/hazelcast-docker/pull/778 ?

Reply:
You're right but vulnerability_scan_subworkflow.yml is used only from master so it didn't hurt but for sake of readability I've reverted it: https://github.com/hazelcast/hazelcast-docker/pull/791/commits/8daf42e4f2bbdf45b7a30a4f2be86355ca5b43a0